src/util.js

```diff
@@ -63,9 +63,12 @@ function varlinkCallError(error) {
  * `varlinkCallChannel()` but allows multiple parallel calls.
  */
 export function varlinkCall(channelOptions, method, parameters) {
-    var channel = cockpit.channel(Object.assign({payload: "stream", binary: true, superuser: "require" }, channelOptions));
+        var channel = cockpit.channel(Object.assign({payload: "stream", binary: true, superuser: "require" }, channelOptions));
+
     return varlinkCallChannel(channel, method, parameters).finally(() => {
-        channel.close()
+        setTimeout(()=>{
+            channel.close()
+        }, 500)
```

Comment:
Why the delay? A hardcoded timeout is almost always a hack, and at least needs an explanation.